integration/mobileiron/README.md

```diff
@@ -1,12 +1,15 @@
 ## Introduction
 
-The scripts here are used to upload built binaries to MobileIron automatically. They can be included in the Jenkins pipeline as part of CI/CD process.
+Include these scripts in your Jenkins pipeline to automatically upload Client App binaries to MobileIron.
 
-## Upload to MobileIron Cloud
+## Prerequisities
+If using macOS, have `grep` installed.
 
-The `cloud_uploadapp.sh` is used to upload apps to MobileIron Cloud.
+## Upload Client App to MobileIron Cloud
```

Comment:
perhaps `Client App Binaries` is better? We are not really upload client apps to MobileIron